packages/python/plotly/plotly/express/_chart_types.py

```diff
@@ -998,6 +998,85 @@ def scatter_mapbox(
 scatter_mapbox.__doc__ = make_docstring(scatter_mapbox)
 
 
+def choropleth_mapbox(
+    data_frame=None,
+    geojson=None,
+    locations=None,
+    color=None,
+    hover_name=None,
+    hover_data=None,
+    custom_data=None,
+    animation_frame=None,
+    animation_group=None,
+    category_orders={},
+    labels={},
+    color_continuous_scale=None,
+    range_color=None,
+    color_continuous_midpoint=None,
+    opacity=None,
+    zoom=8,
+    center=None,
+    mapbox_style=None,
+    title=None,
+    template=None,
+    width=None,
+    height=None,
+):
+    """
+    In a Mapbox choropleth map, each row of `data_frame` is represented by a
+    colored region on a Mapbox map.
+    """
+    return make_figure(
+        args=locals(),
+        constructor=go.Choroplethmapbox,
+        trace_patch=dict(
+            geojson=geojson
+            if not hasattr(geojson, "__geo_interface__")
```

Comment:
this is geopandas-specific code so that you can just say `...geojson=geo_df.geometry...` :)